File: RESTAPI/app/models/ml_data_view.py
from sqlalchemy import Column, String, Float, DateTime, Integer
from app.core.database import Base

class ml_data_view(Base):
    __tablename__ = "ml_data_view"

    site_code = Column(String, primary_key=True, index=True)
    site_name = Column(String)
    site_type = Column(String)
    latitude = Column(Float)
    longitude = Column(Float)
    datetime = Column(DateTime)
    aqi = Column(Integer)
    co = Column(Float)
    no = Column(Float)
    no2 = Column(Float)
    o3 = Column(Float)
    so2 = Column(Float)
    pm2_5 = Column(Float)
    pm10 = Column(Float)
    nh3 = Column(Float)
    temp = Column(Float)
    feels_like = Column(Float)
    pressure = Column(Integer)
    humidity = Column(Integer)
    dew_point = Column(Float)
    wind_speed = Column(Float)
    wind_deg = Column(Integer)
    traffic_flow = Column(String)
    traffic_density = Column(String)

File: DT_App/Models/MLDataView.cs
using System;

namespace DT_App.Models
{
    public class MLDataView
    {
        public string site_code { set; get; }
        public string site_name { set; get; }
        public string site_type { set; get; }
        public double latitude { set; get; }
        public double longitude { set; get; }
        public DateTime datetime { set; get; }
        public int aqi { set; get; }
        public double co { set; get; }
        public double no { set; get; }
        public double no2 { set; get; }
        public double o3 { set; get; }
        public double so2 { set; get; }
        public double pm2_5 { set; get; }
        public double pm10 { set; get; }
        public double nh3 { set; get; }
        public double temp { set; get; }
        public double feels_like { set; get; }
        public int pressure { set; get; }
        public int humidity { set; get; }
        public double dew_point { set; get; }
        public double wind_speed { set; get; }
        public int wind_deg { set; get; }
        public string traffic_flow { set; get; }
        public string traffic_density { set; get; }
    }
}
